src/mcp_analyzer/checkers/security.py

```
@@ -187,6 +193,108 @@ def _check_network_exposure(parsed_url: httpx.URL) -> List[SecurityFinding]:
+        # Check target URL query params for token-like names
+        try:
+            parsed = urlparse(target)
+            query = parsed.query or ""
+            query_hits = []
+            if query:
+                for part in query.split("&"):
+                    if "=" in part:
+                        name, _ = part.split("=", 1)
+                        if key_is_token_like(name):
+                            query_hits.append(name)
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)